app/routes.py

```diff
@@ -1,2 +1,131 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.route_helpers import validate_model
 
+bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                        description=request_body['description'],
+                        radius=request_body['radius']
+                        )
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
+
+@bp.route("", methods=["GET"])
+def read_all_planets():
+
+    name_param = request.args.get("name")
+    description_param = request.args.get("description")
+    radius_param = request.args.get("radius")
+
+    planet_query = Planet.query
+
+    if name_param:
+        planet_query = planet_query.filter_by(name=name_param)
+    if description_param:
+        planet_query = planet_query.filter(Planet.description.ilike(
+            f"%{description_param}%"))
+    if radius_param:
+        # filter planets with a radius greater than query param
+        # How can we give the option of filter lesser than or greater than?
+        # Would we need to prompt the user to include that param and then build
+        # conditionals to check if they want to find greater or less than?
+        planet_query = planet_query.filter(Planet.radius>radius_param)
```

Comment:
👍 Nice job selecting the behaviors for the various filters.

For supporting both greater than and less than, we could include another param, `radius_op` that we check for either `lt` or `gt`, and pick the comparison operator appropriately.